### ISSUE TITLE
What happened with whatsie?

### ISSUE THREAD

**Issue by aditiapratama** [Aluxian/Whatsie#494]
Today I've checked the apps and it's being unpublished, any news about it?

**Comment by hockdudu** [comment]
Hello @aditiapratama, yesterday I've send an e-mail to @Aluxian (The developer), and he told me that he received a Cease and Desist, so he stopped the development from Whatsie and is focusing on other projects now.

**Comment by orangecms** [comment]
Thanks for the clarification. I am very sorry @Aluxian - you have done an awesome job! <3
I highly appreciate the work you put into making WhatsApp a great experience on desktop.

**Comment by llomellamomario** [comment]
Which is a dickish move, since there is no linux desktop app, thus forcing me to have it on a browser. And even if there was I would have to use a light theme, which is not exactly kind to my eyes. I guess I will save the package on my account and ride it for as long as I can.

**Comment by orangecms** [comment]
You can use [nodejs-nativefier](https://www.npmjs.com/package/nativefier) to wrap it, that's what [the package in AUR](https://aur.archlinux.org/packages/whatsapp-web-desktop) is doing.

**Comment by bloodywing** [comment]
Meh, that sucks. I'll keep mine as long as possible too.

**Comment by llomellamomario** [comment]
I can use it, if I knew what I was doing, which I don't. And if I don't know and I consider myself a bit more tech savy than average, I cannot phantom the average user. And that does not accounts for the dark theme. Or the system tray. And probably a lot more other small details that I'm missing but that @Aluxian put into it that made it pleasant to use. Which cycles to the main point, that it still is a dickish move on the Whatsapp team part, send a cease and desist when you don't even have anything to take place.

**Comment by marty90** [comment]
Yes, that's very sad for Whatsapp, it is not a fair behaviour; Aluxian should make some noise about that.

**Comment by shvchk** [comment]
Other alternatives are [Franz](http://meetfranz.com/) and (Rambox)[http://rambox.pro]

**Comment by ecceccecc** [comment]
Using Rambox for now and I can say that it's not as good as Whatsie was. Many bugs, and they're asking everytime for "donation".

**Comment by Leseratte10** [comment]
Isn't Whatsie just basically a web browser which just always visits web.whatsapp.com? Why would WhatsApp be allowed to force someone to stop publishing his web browser just because of the default page it opens? ...

I'll also keep using Whatsie until they finally get a good Linux client done and not just some dumb web app ...

**Comment by CrocodileDandy** [comment]
Where do you get the .deb from exactly now?

**Comment by ecceccecc** [comment]
Here: https://github.com/Aluxian/Whatsie/releases


Em 12/10/2016 13:38, CrocodileDandy escreveu:

**Comment by CRC-Mismatch** [comment]
I'm not entirely sure if there is actual legal backing to support a Cease & Desist, since it's nothing more than a browser. What next, they're gonna send Google, Mozilla, Microsoft, Apple and Opera C&Ds too?
Also, their official "Desktop Client" is not only certainly based off of Aluxian's original WhatsApp-Desktop (it gets detected by Whatsie as an "older WhatsApp-Desktop version"), being itself only a "browser" that accesses WhatsApp-Web, it's also stripped out of many useful features available here...

**Comment by Aluxian** [comment]
WhatsApp did not copy anything from Whatsie.

I do think there could've been some ways to fight the cease and desist, but I did not consider it worth it.

WhatsApp's client might have less features, but it has the most important one -- it's official. Maybe in the future, they'll add the rest. 

--

Alexandru Rosianu
Full-Stack Developer | aluxian.com

**Comment by HoratiuMl** [comment]
That sucks. Whatsie was great, and I will keep using it. It's very sad to see it taken down, and for what? A very feature-less version that doesn't even support my OS (Linux) that I used both on my desktop (Arch) and on my laptop (Ubuntu)... WhatsApp just screwed a lot of people with this move.
